src/Controller/Api/UserController.ts

```diff
@@ -30,4 +32,26 @@ export class UserController {
             res.status(500).json({error: "An uknown error occurred."});
         }
     }
+
+    public async loginUser(req: ValidatedRequest<UserRequestSchema>, res: Response) {
+        try {
+            const {email, password} = req.body;
+            const user = await this.userService.getUserByEmail(email);
+            if (user) {
```

Comment:
I would move this to service too and keep the controller thin. You can throw custom error with status and error message.